Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+ 
+  config.vm.box = "https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-latest.x86_64.vagrant-libvirt.box"
+
+  config.vm.provider :libvirt do |domain|
+    domain.memory = 8192
+    domain.nested = true
+    domain.machine_virtual_size = 30
+    domain.cpus = 8
+    # this will run the VM connected to the same network used testing with a virtual OpenShift in QE
+    domain.management_network_name = "baremetal-0"
+    # Enable KVM nested virtualization
+    domain.nested = true
```

Comment:
dupe?

Reply:
Oh, it is, I will clean it in follow up so we get this in now